apps/rewards/app/components/Panel/PanelManager.js

```diff
@@ -18,12 +18,14 @@ const PANELS = Object.keys(dynamicImport).reduce((obj, item) => {
   return obj
 }, {})
 
-const PanelManager = ({ activePanel = null, onClose, ...panelProps }) => {
-  let panelTitle = activePanel && camel2title(activePanel)
+const PanelManager = ({
+  activePanel = null,
+  onClose,
+  title,
+  ...panelProps
+}) => {
+  let panelTitle = activePanel && (title || camel2title(activePanel))
```

Comment:
This doesn't seem quite right. If `title` was provided, we don't really care if `activePanel` is falsey, right? So maybe more like `title || (activePanel && camel2title(activePanel))`

Reply:
Yes, but no need for parentheses because `&&` takes precedence over `||`